src/graphics/program-lib/chunks/reflectionDpAtlas.frag

```diff
@@ -26,7 +26,7 @@ vec2 getDpAtlasUv(vec2 uv, float mip) {
 }
 
 vec3 calcReflection(vec3 tReflDirW, float tGlossiness) {
-    vec3 reflDir = normalize(cubeMapProject(tReflDirW));
+    vec3 reflDir = normalize(cubeMapProject(tReflDirW)*vec3(-1,1,1));
```

Comment:
Style (operator)

Reply:
I'll do a spacing pass on all the shaders chunks I touched

Reply:
Yeah, unfortunately, we can't run ESLint over the GLSL code unfortunately. So we have to catch these things just with our 👁️👁️

Reply:
shader chunk spacing should all be fixed now - thanks!